File: Interviewbits/PREVIOUS/Min Steps In Infinite Grid.cpp
int Solution::coverPoints(vector<int> &A, vector<int> &B) {
    int count = 0;
    for(int i=0;i<A.size()-1;i++){
        count+=max(abs(A[i+1]-A[i]),abs(B[i+1]-B[i]));
    }
    return count;
}

File: Interviewbits/Level1-Arrays/Min Steps in Infinite Grid.py
class Solution:
	# @param A : list of integers
	# @param B : list of integers
	# @return an integer
	def coverPoints(self, A, B):
        N=len(A)
        minSteps=0
        previous=(A[0],B[0])
        for i in range(1,N):
            current=(A[i],B[i])
            minSteps+=(max(abs(current[0]-previous[0]),abs(current[1]-previous[1])))
            previous=current
            # minSteps+=max(A[i],B[i])
        return minSteps